### ISSUE TITLE
Add must-use plugin using Composer

### ISSUE THREAD

**Issue by Tailzip** [xyu/heroku-wp#96]
Hi,

How can I add a must-use plugin using Composer?

Thanks!

**Comment by xyu** [comment]
Take a look at one of the MU plugins that comes with this repo already, `secure-db-connection`. You will first need to install the plugin via Composer's require:

https://github.com/xyu/heroku-wp/blob/cfe71052c80e7e7c9cfd11061d2c8219d8011768/composer.json#L22

Then add that plugin to the MU plugin custom installer path:

https://github.com/xyu/heroku-wp/blob/cfe71052c80e7e7c9cfd11061d2c8219d8011768/composer.json#L66

**Comment by Tailzip** [comment]
I finally manage to add one (S3 Uploads). After you add the plugin to the MU plugin custom installer path in your `composer.json` file, you also need to create a symlink in `public/wp-content/mu-plugins` that points to main plugin file.

In my case, I ran this command inside `public/wp-content/mu-plugins`: `ln -s s3-uploads/s3-uploads.php s3-uploads.php`.

It would be nice to add a small paragraph in the README :)

**Comment by xyu** [comment]
@Tailzip good point I forgot about the symlink part of that process going to reopen this issue to get that fixed. 🙂
